src/buckets/components/BucketList.tsx

```diff
@@ -63,23 +63,27 @@ const BucketList: FC<Props> = ({
     1
   )
 
-  const paginate = useCallback((page: number) => {
-    const url = new URL(location.href)
-    history.replace(`${url.pathname}?page=${page}`, null)
-    setCurrentPage(page)
-  }, [])
+  const paginate = useCallback(
+    (page: number) => {
+      const url = new URL(location.href)
+      history.replace(`${url.pathname}?page=${page}`, null)
+      setCurrentPage(page)
+    },
+    [history]
```

Comment:
Because `history` is used in the callback body, eslint wants it to be added in the dependency array. Two possible solutions here; disable eslint or add it to the array. I don't see issues with adding it since it won't be causing unnecessary re-renders.